### PR TITLE
Tweak lists and summary to follow row column alignment

### PR DESCRIPTION
**Pull request by hdavidzhu**
image

### CODE REVIEW

File: src/components/main-form.js [hunk 1]
```diff
@@ -568,44 +557,45 @@ class MainForm extends React.Component {
               {longerOption.term} year investment total (
               <b>{formatMoney(longerOption.fv)}</b>).
             </p>
-            <Col>
+          </div>
```
Comment (hdavidzhu) [inline src/components/main-form.js]: 😄 As it turns out, my commit before this was an incorrect use of rows and columns. Rows and columns should always go together, and this content didn't need to be its own row column, so we can just put it on the outside

File: src/components/main-form.js [hunk 2]
```diff
@@ -559,7 +548,7 @@ class MainForm extends React.Component {
           </Card>
         </Col>
         <Col xs='8'>
-          <Summary className='p-4 mr-5'>
+          <div>
```
Comment (hdavidzhu) [inline src/components/main-form.js]: Also, I removed the top margin, b/c if you merge in my navbar changes, it contains enough padding for the whole rest of the page

**Comment by jgozal** [comment]
thanks for taking care of this. I need to run some errands today but I'll try to provide a more detailed review later tonight.  The one thing I noticed is that the separator is not actually centered. You'll be able to see this better if you zoom out a bit:

image

I've never really used `injectGlobal`. Does it change the style in the whole app? or only within the scope of that particular view? I think I'd be more inclined to creating a styled component like <LightBold> that we can import into any view, since <b> by definition has a font-weight of 700 and we might want to use it somewhere else, whether in the view, or in the app. Thoughts here?

File: src/components/main-form.js [hunk 3]
```diff
@@ -50,33 +50,20 @@ option2.term = 30
...
-  .col {
-    margin-left: -2rem;
-  }
-
-  ol {
-    font-size: 15px;
-    margin-right: 2.5rem;
-  }
+// TODO 2019-09-15: Should these be global?
+const Ol = styled('ol')`
```
Comment (jgozal) [inline src/components/main-form.js]: I think we can just leave it as is for now and we can make it global in the future if we need to

**Comment by hdavidzhu** [comment]
@jgozal Ah, the reason why the alignment looks weird is b/c internal content is left aligned, and there's a center divider. This only means that we should limit the size of the summary so it doesn't grow so wide on big screens

**Comment by hdavidzhu** [comment]
> I'm not sure if I like injectGlobal too much as it changes the intended behavior of <b> everywhere without any sort of specificity.

Yep! That's the intent here. I like the 500 weight `b`, and I rather not having other types of `b` in the app. I want to change the default in a way like defining a global theme for the page.

> I'd add a little more space between the text on each side and the divider

How about:

image

> I'm not exactly sure what you meant about the incorrect use of Cols and Rows. Can you elaborate so that I don't make the same mistake?

If there is a `Row`, there should always be a `Col`. Otherwise a row isn't needed:

```
// Good
<Row>
  <Col/>
  <Col/>
</Row>

// Not good
<Row>
  <div/>
  <Col/>
</Row>

// Good
<Row>
  <Col><div/></Col>
  <Col/>
</Row>

// Good
<div/>
<Row>
  <Col/>
</Row>
```

**Pull request merged**